Dockerfile.compute-tools

```diff
@@ -0,0 +1,13 @@
+# First transient image to build compute_tools binaries
+FROM rust:slim-buster AS rust-build
```

Comment:
Not the problem of this particular PR, but interesting to note that here's another place where rustc is not pinned.
Not a big issue to build a submodule, I guess, but still a bit odd.

Reply:
Pinned 1.56.1 everywhere in dockerfiles and put some reference comments to help keeping all these places in sync